tests/IntegrationTests/Client/GetBlockInfo.cs

```diff
@@ -12,6 +12,20 @@ public GetBlockInfo(ITestOutputHelper output) : base(output)
     {
     }
 
+    [Fact]
+    public async Task GivenBlockWithOneTransaction_WhenGetBlockInfo_ThenTransactionCountOne()
+    {
+        // Arrange
+        var blochHashString = this.GetString("blockHashOneTransaction");
+
+
+        // Act
+        var info = await this.Client.GetBlockInfoAsync(new Given(BlockHash.From(blochHashString)));
```

Comment:
I don't understand the point of this test, I would expect this to fail when parsing the blockhash from string and even if this continues I would expect the block to not have any transactions.
What am I missing?

Reply:
The block hash was loaded from configuration and was the block has of a block with one transaction.

Has been removed since none of the other SDK’s have test like this.